docs/examples/basic_conceptual_guide/Part_2-dynamic_batching/Readme.md

```diff
@@ -0,0 +1,240 @@
+
+## Concurrent model execution
+
+The Triton Inference Server can spin up multiple instance of the same model which can process queries in parallel. Triton can spawn instances on the same device (GPU), or a different device on the same node as per the user's specifications. This customizablity is especially useful when considering ensembles which have models with different throughput. Multiple copies of slowers models can be spawned on a separate GPU to allow for more parallel processing. This is enabled via the use of `instance groups` option in a model's configuration.
+
+```
+instance_group [
+  {
+    count: 2
+    kind: KIND_GPU
+    gpus: [ 0, 1 ]
+  }
+] 
+```
+
+Let's take the previous example and discuss the effect of adding multiple model for parallel execution. In this example, instead of having a single model process five queries, two models are spawned. ![Multiple Model Instances](./img/2.PNG)
+
+For a "no dynamic batching" case, as there are model models to execute, the queries are distributed equally. Users can also add [priorities](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#priority) to prioritize or de-prioritize any specific instance group.
+
+When considering the case of multiple instance with dynamic batches enabled, owing to the availablity of another instance, query `B` which arrives with some delay can be executed using the second instance. With some delay allocated, instance 1 gets filled and launched by time `T = X/2` and since queries `D` and `E` stack up to fill up to the maximum batch size, the second model can start inference without any delay. 
+
+The key take away with the above examples is that the Triton Inference Server provides flexiblity with respect to policies related to creating more efficient batching, thus enabling better resource utilzation, resulting in reduced latency and increased throughput.
+
+## Demonstration
+
+This section showcases the use of dynamic batching and concurrent model execution using the example for Part 1 of this series.
+
+### Getting access to the model
+
+Let's use the `text recognition` used in part 1. We do need to make some minor changes in the model, namely making the 0th axes of dynamic shape to enable batching. Step 1, download the Text Recognition model weights. Use the NGC PyTorch container as the environment for the following.
```

Comment:
`0th axes of dynamic shape` seems like a typo. Maybe something along the lines of: making the model's 0th axis have a dynamic shape to enable batching?